rasa/core/policies/ensemble.py

```diff
@@ -53,17 +53,20 @@ def __init__(
         self._check_for_important_policies()
 
     def _check_for_important_policies(self):
-        if "MappingPolicy" not in self.policies:
-            logger.info(
-                "MappingPolicy not included in policy ensemble. Default intents "
-                "'{} and {} will not trigger actions '{}' and '{}'."
-                "".format(
-                    USER_INTENT_RESTART,
-                    USER_INTENT_BACK,
-                    ACTION_RESTART_NAME,
-                    ACTION_BACK_NAME,
-                )
+        for policy in self.policies:
+            if "MappingPolicy" in type(policy).__name__:
```

Comment:
does it work with an `isinstance` instead of type.__name__? i'm thinking something like
```
if not any([isinstance(policy, MappingPolicy) for policy in self.policies]):
```

Reply:
I hope so, that'd be much more readable :D

Reply:
yup, just tried it and it works literally the way you wrote it 👍 
thanks for the suggestion, that's so much better

Reply:
lit 😎